proto/viam/app/package/v1/package.proto

```diff
@@ -0,0 +1,103 @@
+syntax = "proto3";
+
+package viam.app.package.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+
+option go_package = "go.viam.com/api/app/package/v1";
+
+service PackageService {
+  // CreatePackage uploads a package's contents and information to the cloud
+  rpc CreatePackage(stream CreatePackageRequest) returns (CreatePackageResponse) {
+    option (google.api.http) = {
+      post: "/viam/api/v1/service/datamanagement/create"
+    };
+  }
+
+  // DeletePackage receives an authenticated request with the package name and array of versions and removes the associated packages
```

Comment:
`// DeletePackage removes the given package versions.`

Reply:
done!